brightStars_DM_sprint_Feb2020/BeyondRadius.ipynb

```diff
@@ -0,0 +1,422 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Study (false) detections around bright star masks\n",
+    "The radius of current HSC bright star masks, made by Andy Goulding, are selected by comparing the density of detections in increasingly large annuli around each bright stars with the average detection density. \n",
+    "\n",
+    "Still, one of the identified problems with the HSC Public Data Release 2 is that false detections sometimes occur around the edge of these masks. The aim of this notebook is thus to provide a plot of the number of detections as a function of increasing radius, around bright stars and beyond the edge of their mask."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Options\n",
+    "One might want to change the two cells below.\n",
+    "\n",
+    "The first one allows one to choose the path to the Butler repo we will read in, filter (note it only changes masks read in*, since we read detections from merged catalog), tracts, and binning options for the annuli.\n",
+    "\n",
+    "The rerun in `/project/mschmitz/hscRerun/mynobg` is a subset of the 25 central patches in tract 9697, and had local background subtraction turned off.\n",
+    "\n",
+    "Normally, we'd select bright objects to study from an outside catalog (Gaia). But for now, we are just reading in the actual mask region files, which means we have to jump through a few extra hoops.\n",
+    "If we want to look at whole tracts, we can just set `reselectBrightObjects` to `False`; if further selection of bright objects is needed, pass on a string.\n",
```

Comment:
In this case, yes. If a bright star falls on the edge of the sky region you're looking at, it will have a mask, but you might want not to use it here, since the annuli around it will be truncated and you will miss a bunch of detections.